chapters/en/slides/module7/module7_01.Rmd

```diff
@@ -0,0 +1,310 @@
+You can think of it like this:
+
+```
+alt.condition(check-this, if-true-do-this, if-false-do-this)
+```
+
+In this plot we check if our selection `brush` is on the points,
+and if that is true we use the `Origin` dataframe column to color the points,
+otherwise we use the value `'lightgray'`.
+
+`alt.value` is a help function for
+when we want to use a single value for all the data points,
+rather than values from a column in the dataframe.
+
+You can click to drag the selection around 
+and to clear it you can either double click the selection
+or click once outside the selected region.
+
+---
+
+## Linking selections between charts helps exploring observations across more columns
+
+```{python}
+brush = alt.selection_interval()
+ 
+points = (alt.Chart(cars).mark_circle().encode(
+    alt.X('Horsepower', title='Enging power (hp)'),
+    alt.Y('Miles_per_Gallon', title='Fuel efficiency (miles / gal)'),
+    color=alt.condition(brush, 'Origin', alt.value('lightgray')))
+.add_selection(brush))
+
+points | points.encode(y='Weight_in_lbs', x='Acceleration')
+```
+
+Notes:
+In this slide we have plotted `Horsepower` against `Miles_per_Gallon` 
+and `Acceleration` against `Weight_in_lbs`.
+We can see how the data is spread out in both charts,
+but we don't know how the points are related between the two charts.
+
+For example,
+we might wonder if the cars with really powerful engines
+and low mileage
+are the ones with low or high acceleration and weight.
+There is not way to tell this from just looking at these two chart
+but by highlighting points in one of the charts,
+the same observations will be highlighted in the other chart.
+
+Now we can answer our question!
+Our highlight shows that the cars with high engine power and low mileage
+are largely heavy cars with slow acceleration.
+Importantly,
+the relationships of other observations can also be explored in the plot
+without us having to change anything.
+
+As you can see in this slide
+selections are automatically linked between plots
+that are created from the same base chart,
+so we don't need to add any code to link the selection between the charts
+in this case.
+
+---
+
+## Multiple selections can be used to highlight points
+
+```{python}
+brush = alt.selection_interval(resolve='intersect')
+ 
+points = (alt.Chart(cars).mark_circle().encode(
+    alt.X('Horsepower', title='Enging power (hp)'),
+    alt.Y('Miles_per_Gallon', title='Fuel efficiency (miles / gal)'),
+    color=alt.condition(brush, 'Origin', alt.value('lightgray')))
+.add_selection(brush))
+
+points | points.encode(y='Weight_in_lbs', x='Acceleration')
+```
+
+Notes:
+In the previous slide we only had one selection,
+if we started to select in the right hand plot,
+the chart to the left would clear its selection.
+
+We could modify this behavior so that there are separate selections
+in the the two charts and only points that fall within
+the intersection or union of both the selections are highlighted
+as in this slide.
```

Comment:
```suggestion
In the previous slide we only had one selection possible at a time.
For example, if we started to first select observations in the chart
on the left, and then started to make another selection in the chart 
on the right hand plot,
the chart to the left would clear its selection.

We can modify this behaviour so that there are separate selections
in the the two charts and only points that fall within
the intersection or union of both the selections are highlighted
as in this slide.
```

Reply:
Can you explain the second paragraph here a bit further, and explain why this might be useful? Perhaps give an example?